README.md

```diff
@@ -1,18 +1,26 @@
 # RTI BioCloud Docker Library
 
-Welcome to the RTI BioCloud Docker Library! This repository serves as a central location for Docker images used by RTI's cloud-based bioinformatics toolkit.
+Welcome to the RTI BioCloud Docker Library! This repository serves as a central location for building and cataloging the Docker images used by RTI's cloud-based bioinformatics toolkit.
 
 ## Repository Structure
 
 The Dockerfiles are organized in the following structure: `<tool-name>/<tool-version>/Dockerfile`. Each tool has its own directory containing the Dockerfile and related files.
-If a software tool does not have a version, with a custom script for example, use `v1` for the tool-version folder.
-
+If a software tool does not have a version, with a custom script for example, use `v1` for the tool-version folder (e.g., RTI_tool/v1/Dockerfile).
```

Comment:
Also, all_lowercase naming convention for any software tool. so this would be:

`rti_tool/v1/Dockerfile`

Reply:
Can you add the convention for all lowercase and only using underscore as a checklist item below? alternatively in the Repository Structure section.

Reply:
Added this convention to the repository structure section.